fontc/src/config.rs

```diff
@@ -39,32 +39,25 @@ impl Config {
     ///
     /// This serializes the configuration if necessary, and returns any existing
     /// incremental state for this run if it exists.
-    //FIXME: this returns a Result but it looks like we never actually return
-    //an error?
-    pub fn init(&self) -> Result<Input, io::Error> {
+    pub fn init(&self) -> Result<Input, Error> {
         let config_file = self.file();
 
         let ir_paths = IrPaths::new(&self.args.build_dir);
         let ir_input_file = ir_paths.ir_input_file();
         if self.has_changed(&config_file) {
             log::info!("Config changed, generating a new one");
             if ir_input_file.exists() {
-                fs::remove_file(ir_input_file).expect("Unable to delete old ir input file");
+                fs::remove_file(ir_input_file)?;
```

Comment:
IIRC this doesn't actually report which file so perhaps use `Error::FileExpected` which does?